src/systems/userinput/paths.js

```diff
@@ -12,12 +12,15 @@ paths.actions.boost = "/actions/boost";
 paths.actions.startGazeTeleport = "/actions/startTeleport";
 paths.actions.stopGazeTeleport = "/actions/stopTeleport";
 paths.actions.spawnPen = "/actions/spawnPen";
+paths.actions.removePen = "/actions/removePen";
```

Comment:
Why is there `removePen` and `pen.remove`?

Reply:
oops, forgot to remove this one